### PR TITLE
Koch Curve Implementation in Python

### PR DESCRIPTION
**Pull request by Subhajit135**
Tried to create snowflake fractals using koch curve in python using turtle programming.

Thank you for your contribution. Please provide the details requested below.

## ISSUE NUMBER

Please provide a link to the issue this pull request addresses.

## SHORT DESCRIPTION

Please provide a short description of the changes.

## TESTING

Please provide the necessary steps to test the solution.

### CODE REVIEW

**Comment by prateekiiest** [comment]
why did you close your previous PR?

**Comment by Subhajit135** [comment]
It was all messy because i PR 3 projects in a single PR. Now i'm thinking of doing this one by one

**Comment by prateekiiest** [comment]
That's good and modular @Subhajit135

**Comment by Subhajit135** [comment]
Thanks, so this PR is going to be merged na?

**Comment by prateekiiest** [comment]
yeah sure if the review is positive

File: Koch Curve/koch curve.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from turtle import *
+
+#function to create koch snowflake or koch curve
+def snowflake(lengthSide, levels):
+    if levels == 0:
+        forward(lengthSide)
+        return
+    lengthSide /= 3.0
+    snowflake(lengthSide, levels-1)
+    left(60)
```
Comment (prateekiiest) [inline Koch Curve/koch curve.py]: could you comment this part

**Comment by Subhajit135** [comment]
okay thanks.

File: Koch Curve/koch curve.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#importing the libraries
+#turtle standard graphics library for python
+from turtle import *
```
Comment (prateekiiest) [inline Koch Curve/koch curve.py]: since your importing turtle, please mention in the requirements,txt which version you are using

**Comment by prateekiiest** [comment]
can you please post a screen shot of your output?

**Comment by Subhajit135** [comment]
Since its a fractal so i have added a video to explain how the out works.

File: Koch Curve/koch curve.py (hunk 2 above)
Comment (Subhajit135) [reply Koch Curve/koch curve.py]: yeah sure.

File: Koch Curve/koch curve.py (hunk 1 above)
Comment (Subhajit135) [reply Koch Curve/koch curve.py]: on it

**Comment by prateekiiest** [comment]
That's great, can you upload that in this comment?

**Comment by Subhajit135** [comment]
Its seems that i cant add video in this comment so here's the google drive link of the video.

https://drive.google.com/open?id=1pTmBX7GsVqcS4Tj8F69Je1KYJQBZnpt4

File: Koch Curve/koch curve.py (hunk 2 above)
Comment (Subhajit135) [reply Koch Curve/koch curve.py]: turtle is an inbuilt module in python so should i only mention the python version or something else?

Comment (prateekiiest) [reply Koch Curve/koch curve.py]: no need to add now. I will check it later.
Merging this

**Comment by prateekiiest** [comment]
Add description in the README

**Pull request merged**